specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -6215,19 +6215,28 @@
           },
           {
             "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "integrationServiceEnvironmentManagedApi",
+            "description": "The integration service environment managed api.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApi"
```

Comment:
The [`contentLink`](https://github.com/Azure/azure-rest-api-specs/pull/16181/files#diff-5be6fa16a8d0e6a764bc6366111e3ae6f4bed4eeed135f257a16c431aa5be280R9479) under "IntegrationServiceEnvironmentManagedApi" is read-only. I assume `contentLink` is writeable, right?